plugins/menu.js

```
@@ -27,61 +29,61 @@
+    pattern: 'list',
+    public: true,
+    desc: 'Show All Commands',
+    dontAddCommandList: true,
+  },
+  async (message) => {
+    let cmdsList = 'Command List\n\n';
+    let cmdList = [];
+    let cmd, desc;
+    commands.map((command) => {
+      if (command.pattern) cmd = command.pattern.toString().split(/\W+/)[1];
+      desc = command.desc || false;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)